CHANGES.rst

```diff
@@ -1,6 +1,18 @@
 Changes
 =======
 
+Version 0.7.1 (2020-11-10)
+--------------------------
+
+- Users:
+    - Adds support for specifying ``htcondor_max_runtime`` and ``htcondor_accounting_group`` for HTCondor compute backend jobs.
+    - Fixes restarting of Yadage and CWL workflows.
+    - Fixes REANA <-> GItLab synchronisation for projects having additional external webhooks.
+    - Changes ``ping`` command output to include REANA client and server version information.
+- Developers:
+    - Fixes conflicting ``kombu`` installation requirements by requiring Celery version 4.
+    - Changes ``/api/you`` endpoint to include REANA server version information.
```

Comment:
Here we could mention Travis->GA move if we want: (?)

- Changes continuous integration platform from Travis CI to GitHub Actions.